assets/tests/components/propertiesPanel/blockWaivers.test.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react"
+import renderer from "react-test-renderer"
+import BlockWaivers, * as blockWaivers from "../../../src/components/propertiesPanel/blockWaivers"
+import { BlockWaiver } from "../../../src/realtime"
+
+const {
+  CurrentFuturePastType,
+  currentFuturePastClass,
+  currentFuturePastTitle,
+  currentFuturePastType,
+  formatTimeOfDay,
+  hours12,
+  nowTimeOfDay,
+} = blockWaivers
+
+describe("BlockWaivers", () => {
+  test("renders", () => {
+    const blockWaiver: BlockWaiver = {
+      startTime: 18300,
+      endTime: 45480,
```

Comment:
Without mocking `now`, will this snapshot change depending on when you run the test?

Reply:
No, if you look at the snapshot the time is the absolute start time and end time provided in the block waiver. (Also, it's easy to know if you need to mock now because running the tests a second time—including through Semaphore—will always fail.)